contracts/BIFI/strategies/Stellaswap/StrategyStellaswapDualNonNativeLP.sol

```diff
@@ -0,0 +1,323 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+import "../../interfaces/common/IUniswapRouter.sol";
+import "../../interfaces/common/IUniswapV2Pair.sol";
+import "../../interfaces/traderjoe/IMasterChef.sol";
+import "../Common/StratManager.sol";
+import "../Common/FeeManager.sol";
+
+/**
+ * Used to compound StellaDistributorV2
+ */
+contract StrategyStellaswapDualNonNativeLP is StratManager, FeeManager {
+    using SafeERC20 for IERC20;
+    using SafeMath for uint256;
+
+    // Tokens used
+    address public native;
+    address public output;
+    address public secondOutput;
+    address public want;
+    address public lpToken0;
+    address public lpToken1;
+
+    // Third party contracts
+    address public chef;
+    uint256 public poolId;
+
+    bool public harvestOnDeposit;
+    uint256 public lastHarvest;
+
+    // Routes
+    address[] public outputToNativeRoute;
+    address[] public secondOutputToNativeRoute;
+    address[] public nativeToLp0Route;
+    address[] public nativeToLp1Route;
+
+    /**
+     * @dev Event that is fired each time someone harvests the strat.
+     */
+    event StratHarvest(address indexed harvester, uint256 wantHarvested, uint256 tvl);
+    event Deposit(uint256 tvl);
+    event Withdraw(uint256 tvl);
+
```

Comment:
missing ChargedFees event, see [CommonChefLP](https://github.com/beefyfinance/beefy-contracts/blob/master/contracts/BIFI/strategies/Common/StrategyCommonChefLP.sol) for implementation

Reply:
fixed